ui/src/app/pages/homepage/data-access-requirements.tsx

```diff
@@ -0,0 +1,506 @@
+    width: '593px',
+    borderRadius: '0.2rem',
+    backgroundColor: colors.white,
+    border: '1px solid',
+    borderColor: colors.accent,
+  },
+  moduleIcon: {
+    marginLeft: '0.2em',
+    marginRight: '1em',
+  },
+  activeModuleText: {
+    color: colors.primary,
+  },
+  inactiveModuleText: {
+    opacity: '0.5',
+  },
+  moduleDate: {
+    opacity: '0.5',
+    fontSize: '12px',
+  },
+  nextElement: {
+    marginLeft: 'auto',
+  },
+  nextText: {
+    marginLeft: 'auto',
+    alignSelf: 'center',
+    paddingRight: '0.5em',
+  },
+  nextIcon: {
+    fontSize: '18px',
+    color: colors.white,
+    background: colors.success,
+    paddingRight: '0.2em',
+    paddingLeft: '0.2em',
+    alignSelf: 'center',
+  },
+  refreshButton: {
+    height: '25px',
+    width: '81px',
+    fontSize: '10px',
+    borderRadius: '3px',
+    marginLeft: 'auto',
+  },
+  refreshIcon: {
+    fontSize: '18px',
+    paddingRight: '4px',
+  },
+});
+
+// in display order
+const rtModules = [
+  AccessModule.TWOFACTORAUTH,
+  AccessModule.RASLINKLOGINGOV,
+  AccessModule.ERACOMMONS,
+  AccessModule.COMPLIANCETRAINING,
+];
+
+// TODO
+const ctModules = [];
+
+const duccModule = [
+  AccessModule.DATAUSERCODEOFCONDUCT,
+];
+
+// in display order
+// visible for test
+export const allModules: AccessModule[] = [
+  ...rtModules,
+  ...ctModules,
+  ...duccModule,
+];
+
+const LoginGovTooltip = () => <TooltipTrigger
+    content={'For additional security, we require you to verify your identity by uploading a photo of your ID.'}>
+  <InfoIcon style={{margin: '0 0.3rem'}}/>
+</TooltipTrigger>;
+
+const moduleLabels: Map<AccessModule, JSX.Element> = new Map([
+  [AccessModule.TWOFACTORAUTH, <div>Turn on Google 2-Step Verification</div>],
+  [AccessModule.RASLINKLOGINGOV, <div>Verify your identity with Login.gov <LoginGovTooltip/></div>],
+  [AccessModule.ERACOMMONS, <div>Connect your eRA Commons account</div>],
+  [AccessModule.COMPLIANCETRAINING, <div>Complete <AoU/> research Registered Tier training</div>],
+  [AccessModule.DATAUSERCODEOFCONDUCT, <div>Sign Data User Code of Conduct</div>],
+]);
+
+// this function does double duty:
+// - returns appropriate text for completed and bypassed modules and null for incomplete modules
+// - because of this, truthy return values indicate that a module is either complete or bypassed
+const bypassedOrCompletedText = (status: AccessModuleStatus) => {
+  const {completionEpochMillis, bypassEpochMillis}: AccessModuleStatus = status || {};
+  const userCompletedModule = !!completionEpochMillis;
+  const userBypassedModule = !!bypassEpochMillis;
+
+  return cond(
+      [userCompletedModule, () => `Completed on: ${displayDateWithoutHours(completionEpochMillis)}`],
+      [userBypassedModule, () => `Bypassed on: ${displayDateWithoutHours(bypassEpochMillis)}`],
+      // return nothing if there's no text
+    () => null
+  );
+};
+
+const handleTerraShibbolethCallback = (token: string, spinnerProps: WithSpinnerOverlayProps) => {
```

Comment:
ras-callback and nih-callback are routed to here when the env flag is true.  Adapted this code from Homepage.